app/users/components/ConnectedCalendars.tsx

```
@@ -1,12 +1,25 @@
 import { ConnectedCalendar } from "@prisma/client"
-import { Link } from "blitz"
+import { useMutation } from "blitz"
 import { Button, Table } from "react-bootstrap"
+import deleteConnectedCalendar from "../mutations/deleteConnectedCalendar"
 
 interface ConnectedCalendarsProps {
   calendars: Omit<ConnectedCalendar, "encryptedPassword">[]
 }
 
 const ConnectedCalendars = (props: ConnectedCalendarsProps) => {
+  const [deleteCalendar] = useMutation(deleteConnectedCalendar)
+
+  const submitDeletion = async (calendarId: number) => {
+    try {
+      const calendar = await deleteCalendar(calendarId)
+      const index = props.calendars.findIndex((calendarEntry) => calendarEntry.id === calendarId)
+      props.calendars.splice(index, 1)
+    } catch (error) {
+      alert(error)
```

This is a little defensive. We should instead let it throw and setup smth like Sentry to be notified abouts these errors.